core/local/atom_watcher.js

```diff
@@ -0,0 +1,86 @@
+/* @flow */
+
+const fse = require('fs-extra')
+const Promise = require('bluebird')
+
+const checksumer = require('./checksumer')
+const logger = require('../logger')
+const LinuxSource = require('./layers/linux')
+const Identifier = require('./layers/identifier')
+const ChecksumLayer = require('./layers/checksum')
+const Dispatcher = require('./layers/dispatcher')
```

Comment:
`ChecksumLayer` has a suffix, others don't. Standardize?

Reply:
Well, I prefer to have no suffix. But we already have a lot of stuff called checksum and checksumer and I don't want to add confusion to that.